pipeline/gene_to_nissl.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script that maps Gene Expression to Nissl."""
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import pathlib
+import sys
+from pathlib import Path
+
+import nrrd
+import numpy as np
+from skimage.color import rgb2gray
+
+from atlannot.ants import register, transform
+
+# Initialize the logger
+logger = logging.getLogger("gene-to-nissl")
+DATA_FOLDER = pathlib.Path(__file__).resolve().parent.parent / "data"
+
+
+def parse_args():
+    """Parse arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--gene-path",
```

Comment:
BTW do we not care about the thresholded expression images?

The `download_gene.py` does download them. But I guess one can just call this script twice with different input arrays.

Reply:
I thought about dedicating a PR for handling the thresholded expression images (adding a argument in the CLI, registering them and so on) and also maybe discuss exactly what Dimitri and Csaba wants about that (should we also do interpolation for those images), and so on. There is a point for this in the (huge) to do list.